docker-compose.yml

```diff
@@ -9,6 +8,8 @@ services:
             POSTGRES_DB: notification_service
         ports:
             - 5434:5432
+#        networks:
+#            - helsinki
```

Comment:
Is there a real reason why the network is commented out? The same "helsinki" network is in use in every service.